mws/data_mws_credentials.go

```diff
@@ -0,0 +1,30 @@
+package mws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceMwsCredentials() *schema.Resource {
+	type mwsCredentialsData struct {
+		Ids map[string]string `json:"ids,omitempty" tf:"computed,slice_set"`
```

Comment:
`slice_set` isn't necessary here. It's only for lists

Reply:
Removed slice_set